app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for character in phrase:
+        if character.lower() == letter.lower():
+            counter += 1
+
+    return counter
```

Comment:
try to write in one line

Reply:
Also you close your pull request 
Please reopen it

Reply:
Pull request was reopened. 
The solution was written in one line.